examples/src/main/scala/com/http4s/rho/swagger/demo/Main.scala

```diff
@@ -1,15 +1,28 @@
 package com.http4s.rho.swagger.demo
 
-import org.http4s.rho.swagger.SwaggerSupport
 import org.http4s.Service
+import org.http4s.rho.swagger.SwaggerSupport
 import org.http4s.server.blaze.BlazeBuilder
+import org.http4s.server.{Server, ServerApp}
+import org.log4s.getLogger
+
+import scalaz.concurrent.Task
+
+object Main extends ServerApp {
+  private val logger = getLogger
+
+  val port = Option(System.getenv("HTTP_PORT"))
```

Comment:
Can't take the credit for that one, it was lifted directly from the existing Hal example.  Glad you like it though.